handlers.go

```diff
@@ -69,7 +66,7 @@ func (dht *IpfsDHT) handleGetValue(ctx context.Context, p peer.ID, pmes *pb.Mess
 	resp.Record = rec
 
 	// Find closest peer on given cluster to desired key and reply with that info
-	closer := dht.betterPeersToQuery(pmes, p, CloserPeerCount)
+	closer := dht.betterPeersToQuery(pmes, p, dht.bucketSize)
```

Comment:
let's avoid changing the number of peers we respond with. that will have bad impacts on the bandwidth usage